modules/metastore/tests/src/Unit/Reference/ReferenceLookupTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+// namespace Drupal\Tests\metastore\Unit\Reference;
+
+// use Drupal\Core\Cache\CacheTagsInvalidator;
+// use Drupal\Core\Config\ConfigFactory;
+// use Drupal\Core\Config\ImmutableConfig;
+// use Drupal\Core\Queue\QueueFactory;
+// use Drupal\Core\Queue\QueueInterface;
+// use Drupal\metastore\NodeWrapper\NodeDataFactory;
+// use Drupal\metastore\Reference\OrphanChecker;
+// use Drupal\metastore\Reference\ReferenceLookup;
+// use Drupal\metastore\Storage\Data;
+// use Drupal\metastore\Storage\DataFactory;
+// use Drupal\metastore\Storage\NodeData;
+// use MockChain\Chain;
+// use MockChain\Options;
+// use PHPUnit\Framework\TestCase;
+// use Symfony\Component\DependencyInjection\ContainerInterface;
+
+// /**
+//  * @todo Finish these tests.
+//  */
+// class OrphanCheckerTest extends TestCase {
```

Comment:
If these tests need to be added later, then I think a separate branch/PR should be created for them and referenced in this PR's ticket as opposed to just commenting out the progress that's been made.

Reply:
Other than this, I think the code looks good.
This is more of a suggestion. I just want to make sure we don't get in the habit of pushing files which are never used and forgotten to the code base.

Reply:
Yeah that's very fair, I'll split it off into its own PR draft.